examples/rust/wallet-recovery/src/main.rs

```diff
@@ -1,73 +1,74 @@
-// detailed documentation for this code can be found at https://bitcoindevkit.github.io/book-of-bdk/cookbook/quickstart/
```

Comment:
I don't think we need a wallet recovery example in this PR, because the code presented is the same as simply creating a wallet with a given mnemonic.